### ISSUE TITLE
Large Image Sizes and PNG Format Causing Performance Issues.

### ISSUE THREAD

**Issue by ashparshp** [layer5io/layer5#5854]
**Current Behavior**
The current images used in the project are of heavy size, and many are in PNG format. This is causing performance issues, especially with page loading times. To improve performance, we need to:

**Desired Behavior**

Convert the PNG images to the WebP format, which offers better compression and smaller file sizes.
Reduce the size of other heavy images without compromising too much on quality.
Tasks:

- [ ]  Identify all PNG images in the project.
- [ ]  Convert PNG images to WebP format.
- [ ]  Reduce the size of large images while maintaining acceptable quality.
- [ ]  Test the website to ensure that the images load correctly and that performance is improved.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

**Comment by Mandeep56Singh** [comment]
@Ashparshp  I would like to work on this

**Comment by Ashparshp** [comment]
Thank you @Mandeep56Singh, please go ahead.

**Comment by Mandeep56Singh** [comment]
@Ashparshp while running make site 
my build got stuck .
![Screenshot (449)](https://github.com/user-attachments/assets/6a4a9cbe-6ed9-43d5-8dcc-6dc3dd3839a4)

my system
windows 11
system type : 64-bit operating system, x64-based processor
processor 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz   2.40 GHz

I have searched about this issue, a lot of people having x64 have this issue. 
do you have any idea how to resolve it ?

**Comment by Ashparshp** [comment]
@Mandeep56Singh, It happens. Try running `make site` in the terminal, not in the integrated VS Code terminal.

**Comment by Mandeep56Singh** [comment]
no luck, I tried windows terminal as well as bash

**Comment by Ashparshp** [comment]
It takes some time, please wait for it. It works fine in windows.

**Comment by Ashparshp** [comment]
Otherwise use GitHub Codespace.

**Comment by Ashparshp** [comment]
@Mandeep56Singh, Solved the setup?

**Comment by Mandeep56Singh** [comment]
Sorry bro, I am having my MST's but I will resolve this issue. 
I will do update you tomorrow.
Thanks.

**Comment by Mandeep56Singh** [comment]
@Ashparshp  about codespaces sometimes it works sometimes not,
however I thought I would first finish the job then find a way to run it
here how I am approaching this issue
- I would first optimized the png, 
- then would convert it to webp 
- then further optimization.

however for maximum compression, we can reduce image dimension. most of image have dimension exceeding 1920 * 1080
if we reduce their dimension it would be very low size image.
like converting a 1920 * 1080 to 800 * 400 would lead to 85% size reduction resulting to greater higher loading speed in website
if I have the info what the required maximum for image I can test it.
I am step 1 now.
I would appreciate your response.

**Comment by Ashparshp** [comment]
@Mandeep56Singh, a few things to keep in mind:
For the members' profiles, we can use low-quality images, but in certain sections, we need to maintain a higher quality.
Also, some WebP images have large file sizes that can be reduced.

Feel free to go ahead and raise a PR and we'll discuss about the PR, in Monday website call..

**Comment by Mandeep56Singh** [comment]
@Ashparshp right now, we have two different format but same image in 
src\assets\images\service-mesh-icons\open-service-mesh\horizontal\black\openservicemesh-horizontal-black.svg
src\assets\images\service-mesh-icons\open-service-mesh\horizontal\black\openservicemesh-horizontal-black.png

src\assets\images\service-mesh-icons\open-service-mesh\horizontal\white\openservicemesh-horizontal-white.svg
src\assets\images\service-mesh-icons\open-service-mesh\horizontal\white\openservicemesh-horizontal-white.png


the png is of very size resolution 18K * 5K 
should I proceed on deleting the png since we can use svgs

**Comment by Ashparshp** [comment]
@sudhanshutech, take a look please!!

**Comment by Ashparshp** [comment]
@Mandeep56Singh, for now we can avoid this deletion, and you can ask further in the Monday website call. Please join us if you can.

Please go ahead and raise a PR for the rest!

**Comment by jason2000-cpu** [comment]
I would like to work on this issue

**Comment by Rajesh-Nagarajan-11** [comment]
@jason2000-cpu  Thank you for picking this

**Comment by jason2000-cpu** [comment]
@Rajesh-Nagarajan-11 I am converting the .png images to .webp . should I delete the .img images after converting?

**Comment by saurabhraghuvanshii** [comment]
> [@Rajesh-Nagarajan-11](https://github.com/Rajesh-Nagarajan-11) I am converting the .png images to .webp . should I delete the .img images after converting?

yes

**Comment by Rajesh-Nagarajan-11** [comment]
@jason2000-cpu   , yes try to implement this without compromising the quality of images
